components/transport_profiles/app/models/transport_profiles/processes/push_nfp_commission_statement.rb

```diff
@@ -0,0 +1,63 @@
+module TransportProfiles
+  class Processes::PushNfpCommissionStatement < Processes::Process
+
+    def initialize(report_file_name, gateway)
+      super("Distribute system generated reports to defined data stores", gateway)
```

Comment:
Need to adjust description above.

Reply:
something like upload commission statements to S3